### ISSUE TITLE
Navigation is not possible in TabStrip with NVDA

### ISSUE THREAD

**Issue by nelito987** [telerik/kendo-ui-core#7606]
### Bug report

When NVDA is turned on the navigation in TabStrip is not working.

### Reproduction of the problem
1. Open the TabStrip demos - https://demos.telerik.com/kendo-ui/tabstrip/index
2. Try to navigate between the tabs using the arrow keys.

### Current behavior
The selection of the item is not changed.

### Expected/desired behavior
It should be possible to navigate between the tabs in the TabStrip when NVDA is turned on.

### Environment

* **Kendo UI version:** 2023.3.1114
* **Browser:** [all]
